sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -915,7 +916,7 @@ def _settle_message(
                                                    dead_letter_reason=dead_letter_reason,
                                                    dead_letter_error_description=dead_letter_error_description)()
                     return
-                except RuntimeError as exception:
+                except Exception as exception:  # pylint: disable=broad-except
```

Comment:
My only fear from this is we fall back to mgmt for exceptions that are "legitimate, non-retriable" exceptions.  Any way we can scope this further?

Reply:
revert back to catch RuntimeError as uamqp library is only raising RuntimeError in case of failure.